plugins/modules/ipa_pwpolicy.py

```diff
@@ -93,9 +108,15 @@
       historylength: '16'
       minclasses: '4'
       priority: '10'
+      minlength: '6'
       maxfailcount: '4'
       failinterval: '600'
       lockouttime: '1200'
+      gracelimit: '3'
```

Comment:
(I can see here that other options already seem to have the same problem, but maybe it's better to start doing it better now?)

Reply:
Done